wrapper.js

```diff
@@ -207,7 +207,7 @@ function setupMethods (soljson) {
       var url = 'https://ethereum.github.io/solc-bin/bin/soljson-' + versionString + '.js';
       https.get(url, function (response) {
         if (response.statusCode !== 200) {
-          cb('Error retrieving binary: ' + response.statusMessage);
+          cb(new Error('Error retrieving binary: ' + response.statusMessage));
```

Comment:
Cool, the new semistandard can catch this?

Reply:
Yes, it can! When I bumped the version of semistandard, it said: `Unexpected literal in error position of callback.` so I fixed that.